.github/workflows/cd.yaml

```diff
@@ -68,12 +68,16 @@ jobs:
       matrix:
         base:
           - php/8.2/fpm-nginx
+          - php/8.2/fpm-apache
           - php/8.3/fpm-nginx
+          - php/8.0/cli
```

Comment:
How come the PHP 8.0 lines were added in this PR without a respective Dockerfile? Can we sort this please - it has broken the CI.

@CWRahien @jcrozier @barkerl

Reply:
That looks to be the php8.0 VOL cli image if i had to guess. It looks like this will have been a timing/rebasing issue after those were removed from the VOL side.

I think these can probably safely be removed, the CI workflows weren't failing prior to merge.